shadowsocks-libev: update AEAD cipher names

leonghui opened the pull request:
Maintainer: @yousong
Compile tested: mips/ar71xx, DIR-825-C1, r4683-370aacf532
Run tested: mips/ar71xx, DIR-825-C1, r4683-370aacf532

Description:
Updated AEAD cipher names in aead_methods:
- chacha20-ietf-poly1305
- xchacha20-ietf-poly1305 

Reference: https://github.com/shadowsocks/shadowsocks-libev/blob/master/src/aead.c

Signed-off-by: Leong Hui Wong <wong.leonghui@gmail.com>

yousong commented:
Hi, @leonghui

The change looks good, thanks.  Please squash the commitsinto a single one.

I left them out previously because they appear in the `supported_aead_ciphers_mbedtls` as `CIPHER_UNSUPPORTED`, but it looks like they actually depends on constructs in libsodium ;)

yousong commented:
Please also increment PKG_RELEASE` in the package Makefile

yousong commented:
The link in the commit message body will return `Not found`.  Maybe it can be just omitted.

leonghui commented:
Hi @yousong 
Added to PKG_RELEASE and remove the link from the commit message body. Updated the reference in my post above for tracking purposes. Thanks!

yousong commented:
Thanks!
